mail/backup_mailer/src/opnsense/mvc/app/library/OPNsense/Backup/Mailer.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace OPNsense\Backup;
+
+require_once('phpmailer/PHPMailerAutoload.php');
+
+use OPNsense\Core\Config;
+use OPNsense\Backup\MailerSettings;
+
+/**
+ * Class mail backup
+ * @package OPNsense\Backup
+ */
+class Mailer extends Base implements IBackupProvider
+{
+
+    /**
+     * get required (user interface) fields for backup connector
+     * @return array configuration fields, types and description
+     */
+    public function getConfigurationFields()
+    {
+        $fields = array();
+
+        $fields[] = array(
+            "name" => "MailEnabled",
+            "type" => "checkbox",
+            "label" => gettext("Enable"),
+            "value" => null
+        );
+        $fields[] = array(
+            "name" => "Receiver",
+            "type" => "text",
+            "label" => gettext("Receiver Email Address"),
+            "value" => null
+        );
+        $fields[] = array(
+            "name" => "SmtpHost",
+            "type" => "text",
+            "label" => gettext("SMTP Host"),
+            "value" => null
+        );
+        $fields[] = array(
+            "name" => "SmtpPort",
+            "type" => "text",
+            "label" => gettext("SMTP Port"),
+            "value" => null
+        );
+        $fields[] = array(
+            "name" => "SmtpSSL",
+            "type" => "checkbox",
+            "label" => gettext("Use SSL"),
```

Comment:
Use TLS

Reply:
PHPMailer itself already verifies if the server can use TLS and automatically enables it if possible

Reply:
upon further reading I think its indeed the best solution to stop using ssl and "force" tls